Dockerfile.fedora-rawhide

```diff
@@ -1,15 +1,15 @@
 # Clone from the Fedora rawhide image
 FROM registry.fedoraproject.org/fedora:rawhide
 
-MAINTAINER Jan Pazdziora
+MAINTAINER FreeIPA Developers <freeipa-devel@lists.fedorahosted.org>
 
 RUN groupadd -g 288 kdcproxy ; useradd -u 288 -g 288 -c 'IPA KDC Proxy User' -d '/var/lib/kdcproxy' -s '/sbin/nologin' kdcproxy
 RUN groupadd -g 289 ipaapi; useradd -u 289 -g 289 -c 'IPA Framework User' -d / -s '/sbin/nologin' ipaapi
 
-RUN mkdir -p /run/lock && dnf upgrade -y && dnf install -y freeipa-server freeipa-server-dns freeipa-server-trust-ad initscripts && dnf clean all
```

Comment:
Should this be fixed for older containers, too?

Reply:
I don't think I care. We will get the change for Fedora 28 and onwards and it does not really hard older version.

Reply:
Right, that's fine